.travis.yml

```diff
@@ -1,12 +1,29 @@
 language: c
 sudo: false
-compiler:
-- clang
-- gcc
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: linux
+      compiler: clang
+    - os: osx
+      compiler: gcc
+    - os: osx
+      compiler: clang
+addons:
+  apt:
+    sources:
+    # contains CMake 2.8.12
+    - kubuntu-backports
+    packages:
+    - cmake
 script:
-- ./autogen.sh
-- ./configure
-- make distcheck
+  - mkdir build
+  - cd build
+  - cmake --version
+  - cmake -DCMAKE_C_COMPILER=$CC ..
```

Comment:
I think you're correct, but recall adding this explicitly at some point in the past for a reason ... that reason may no longer hold ... for this reason, it's better to remove this line and see if it still works in all cases. It can be revisited and re-added later if it's a problem.